test/blackbox-tests/test-cases/variables-for-artifacts.t/run.t

```diff
@@ -240,7 +240,7 @@ field of a (rule).
   File "dune", line 3, characters 7-16:
   3 |  (deps %{cmo:x2})
              ^^^^^^^^^
-  Error: %{cmo:..} isn't allowed in this position.
+  Error: Module X2 does not exist.
```

Comment:
Looks like another case, similar to `%{read}`.

Reply:
Indeed, added a TODO30 tag here as well.